frontend/src/App.jsx

```
@@ -1,8 +1,18 @@
 import React, { useContext, useMemo } from "react";
+
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from "react-router-dom";
+import { AppContext } from "./context/AppContext";
+
 import { BrowserRouter as Router, Routes, Route, Navigate } from "react-router-dom";
```

Biome (2.3.13)

[error] Shouldn't redeclare 'Router'. Consider to delete it or rename it.

'Router' is defined here:


(lint/suspicious/noRedeclare)